content/applications/inventory_and_mrp/inventory/product_management/product_tracking/package.rst

```diff
@@ -9,32 +9,28 @@ items in bulk.
 
 Packages are commonly used for the following purposes:
 
-#. :ref:`Relocating products from stock <inventory/warehouses_storage/package-use>` to packing or
-   outbound areas for delivery.
-#. :ref:`Shipping to customers <inventory/warehouses_storage/package-type>`, in which package types
-   are configured to align with shipping carriers' size and weight requirements, streamlining the
-   packing process, and ensuring compliance with carrier shipping specifications.
+#. :ref:`Grouping products to move them in bulk <inventory/warehouses_storage/pack>`.
+#. :ref:`Shipping to customers <inventory/warehouses_storage/package-type>`: configure package types
+   to align with shipping carriers' size and weight requirements, streamlining the packing process,
+   and ensuring compliance with carrier shipping specifications.
 #. Storing items in bulk.
 
 *Package use* is a field on the package form in Odoo that is only visible by enabling the *Batch
-Transfers* feature in the settings of the *Inventory* application.
+Transfers* feature (:menuselection:`Inventory --> Configuration --> Settings`).
 
-The default package type is *Disposable*, so this feature is **only** required when packages are
-intended to be used for batch transfers and/or other internal warehouse moves.
+By default, *Package use* is set to *Disposable*. Change this field to *Reusable* **only** when
+configuring packages for :ref:`cluster pickings <inventory/warehouses_storage/cluster-pack>`.
```

Comment:
```suggestion
By default, the *Package Use* field on a packages form is set to *Disposable Box*. Change this field
to *Reusable Box* **only** when configuring packages for :ref:`cluster pickings
<inventory/warehouses_storage/cluster-pack>`.
```
in the 16.4 runbot I am using for this review, it says the Package Use options are as suggested above, with the word "box" after each one. Please double-check to ensure accuracy here. Thanks!

Reply:
I understand not capitalizing both words when simply introducing and defining the field, but when speaking directly about it, I think it's best to use the proper capitalization. And, since there is no screenshot, and you're not directing them a Package form, nor are we expecting the reader to be on a form like that at this point in the doc -- I think your instinct in not using a GUI is correct.